config/blade.php

```diff
@@ -4,6 +4,8 @@
     // How many seconds past compiled file last modified time to recompile the template
     // A value of 0 is always recompile
     // Note: homestead time verses pc time may be off
-    'secondsTemplateCacheExpires' => 300
+    'secondsTemplateCacheExpires' =>  env('STRING_BLADE_CACHE_TIMEOUT') ?? 300,
```

Comment:
env('STRING_BLADE_CACHE_TIMEOUT', 300) ?

Reply:
I changed it.